CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ Entries are listed in reverse chronological order.
 ## 1.0.2
 
 * Updates the library to use the renamed functions in Merlin 1.1.
-* Adds additional validation checks to prevent identity points being used as
+* Adds additional validation checks to prevent identity points from being used as
```

Comment:
This was correct as written. The "prevent...from" idiom is also correct, but the preposition is not required.